### PR TITLE
student received grade, not tutor graded student

### PR DESCRIPTION
**Pull request by garemoko**
fixes jlowe64/moodle-logstore_xapi#24

### CODE REVIEW

**Comment by garemoko** [comment]
@ryansmith94 going for lunch but I'm pretty sure this will pass tests. If it does, it's ready for review and will close #24.

File: tests/AssignmentGradedTest.php [hunk 1]
```diff
@@ -23,12 +23,20 @@ protected function constructInput() {
     }
 
     protected function assertOutput($input, $output) {
-        parent::assertOutput($input, $output);
```
Comment (ryansmith94) [inline tests/AssignmentGradedTest.php]: Why was this removed? I'm assuming it had something to do with line 26.

Comment (ryansmith94) [inline tests/AssignmentGradedTest.php]: Never mind, I can see that's exactly why it was. All good.
